common/build.gradle

```diff
@@ -6,6 +6,19 @@ plugins {
 def artifactName = "honeycomb-opentelemetry-common"
 description = 'Functionality shared across components of the Honeycomb OpenTelemetry distribution.'
 
+test {
+    useJUnitPlatform()
```

Comment:
A little experiment on this branch, I put this in the root `build.gradle`:

```groovy
subprojects {
    // … 
    test {
        useJUnitPlatform()
        reports {
            junitXml.enabled = true
        }
    }
    // … 
}
```

… and removed the `test {}` stanza from the individual subprojects' `build.gradle` files.


1. Test config declared in a single place.
2. Nothing blew up.
  - Projects that don't depend on junit or have tests seem fine with this config declared for them anyway.
3. The JUnit tests still ran.
4. … now with XML output that could get collected and displayed by CI.

```
» find ./ -type f -name \*.xml                                                                                           1 ↵
.//smoke-tests/report.xml
.//common/build/test-results/test/TEST-io.honeycomb.opentelemetry.EnvironmentConfigurationTest.xml
.//common/build/test-results/test/TEST-io.honeycomb.opentelemetry.DistroMetadataTest.xml
.//sdk/build/test-results/test/TEST-io.honeycomb.opentelemetry.OpenTelemetryConfigurationTest.xml
.//sdk/build/test-results/test/TEST-io.honeycomb.opentelemetry.DeterministicTraceSamplerTest.xml
```

Reply:
I thought about adding it to the root grade.build too 👍🏻